stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -147,64 +154,87 @@ internal class PaymentSheetViewModel internal constructor(
             runCatching {
                 stripeIntentRepository.get(args.clientSecret)
             }.fold(
-                onSuccess = ::onPaymentIntentResponse,
+                onSuccess = ::onStripeIntentResponse,
```

Comment:
If we are renaming this function, can we just name it something like processFetchResponse?  Or something that shows it is only called as a result of a fetch.  I get this mixed up with the flowResult all the time, but maybe I am the only one.